schema/data/anchored/v0/classes/USS01.jsonld

```diff
@@ -1,97 +1,48 @@
-      "@type": "schema:Duration"
-    },
-    "regen:ecosystemType": {
-      "@container": "@list"
-    },
-    "regen:permanencePeriod": {
-      "@type": "schema:Duration"
-    },
-    "schema:itemListElement": {
-      "@container": "@list"
-    },
-    "regen:eligibleActivities": {
-      "@container": "@list"
-    },
-    "regen:creditGenerationMethod": {
-      "@container": "@list"
-    }
+    "rfs": "https://framework.regen.network/schema/",
+    "schema": "http://schema.org/"
   },
   "schema:url": "https://www.registry.regen.network/crediting-protocols/era-brazil",
   "schema:name": "Biodiversity Credit Protocol for Umbrella Species Stewardship",
-  "regen:coBenefits": [
+  "rfs:hasCoBenefits": [
     {
-      "@id": "regen:Improved-Wildlife-Habitat",
-      "@type": "regen:ProjectBenefit",
-      "regen:SDGs": [
-        {
-          "@id": "regen:15LifeOnLand"
-        }
+      "rfs:impactType": "IMPROVED_WILDLIFE_HABITAT",
+      "rfs:supportsSDG": [
+        "SDG_15"
       ]
     },
     {
-      "@id": "regen:Improved-Education-And-Outreach",
-      "@type": "regen:ProjectBenefit",
-      "regen:SDGs": [
-        {
-          "@id": "regen:4QualityEducation"
-        },
-        {
-          "@id": "regen:17Partnerships"
-        },
-        {
-          "@id": "regen:8DecentWork"
-        }
+      "rfs:impactType": "IMPROVED_ENVIRONMENTAL_EDUCATION_OUTREACH",
+      "rfs:supportsSDG": [
+        "SDG_4",
+        "SDG_17",
+        "SDG_8"
       ]
     }
   ],
   "schema:description": "The Biodiversity Crediting Protocol for Umbrella Species Stewardship (USS) provides a comprehensive framework for measuring, verifying, and rewarding communities for their conservation of umbrella species—organisms whose protection benefits entire ecosystems and their inhabitants. The protocol measures conservation success through three integrated components: Umbrella Species Health, which tracks population dynamics and distribution; Habitat Quality, which assesses ecosystem composition and function; and a Theory of Change, which translates community knowledge and long-term goals into strategic conservation activities. By combining on-the-ground data collection, remote sensing, and traditional ecological knowledge, the protocol enables scientifically rigorous and transparent verification of conservation outcomes. All ecological data anchors securely to a blockchain, ensuring credible, verifiable proof of impact. The USS credits represent actual implementation costs, directly linking conservation finance to real biodiversity gains while preventing speculative trading. This empowers local communities and land stewards to address critical biodiversity threats and strengthens their essential role in long-term ecosystem stewardship.",
-  "regen:creditingTerm": "P5Y",
-  "regen:ecosystemType": [
-    "Terrestrial"
+  "rfs:hasCreditingPeriod": "P5Y",
+  "rfs:eligibleEnvironmentTypes": [
+    "Terrestrial -- not found in taxonomy"
```

Comment:
This one is interesting, as it's applicable in any non-anthropogenic terrestrial ecosystem. So most of the elements in the taxonomy are applicable and I'm not sure we want to list all of them. Wondering if include another one to indicate that, or just list the ones it's been applied in to-date

Reply:
We can also simply add "Terrestrial" as a generic ecosystem category (similar to how we have "Marine"?).  Not sure if that's valuable. 

If we don't want to add "Terrestrial", I would prefer just only listing the ones that have been used to date.